app/controllers/sessions_controller.rb

```diff
@@ -5,15 +5,16 @@ def create
     user = User.find_by email: params[:session][:email].downcase
     if user&.authenticate params[:session][:password]
       log_in user
+      remember user
       redirect_to user
     else
-      flash[:danger] = t "user_login.invalid_user" 
+      flash[:danger] = t "user_login.invalid_user"
```

Comment:
nếu dùng render thì sử dụng method now để ko bị lỗi load lại trang vẫn hiển thị flash nha em
```suggestion
      flash.now[:danger] = t "user_login.invalid_user"
```